src/kaleidoscope/plugin/LEDControl.h

```diff
@@ -158,8 +100,9 @@ class LEDControl : public kaleidoscope::Plugin {
 
  private:
   static uint16_t syncTimer;
-  static LEDMode *modes[LED_MAX_MODES];
-  static uint8_t mode;
+  static uint8_t mode_id;
```

Comment:
This should be named `mode_id_`.

Reply:
I was tempted to adapt the `LEDControl` class to be in accordance with the style guide but i resisted. The only symbol name change I made was to rename `mode` to `mode_id` because of a name conflict. The rest remains to be done in a separate PR that will clean up `LEDControl`.

Reply:
Okay, but since you changed the name anyway, why not make it compliant now? There's another spot i noticed where `mode_id` is used as the name of a function parameter, and this makes it more confusing.